Formula/blocky.rb

```diff
@@ -0,0 +1,65 @@
+class Blocky < Formula
+  desc "Fast and lightweight DNS proxy as ad-blocker with many features"
+  homepage "https://0xerr0r.github.io/blocky"
+  url "https://github.com//0xerr0r/blocky/archive/refs/tags/v0.20.tar.gz"
+  sha256 "aae5346e9c1ce4b326b9e578939aa26ddca39338d79d0ddb3eb079ae7a949e87"
+  license "Apache-2.0"
+  head "https://github.com/0xerr0r/blocky.git", branch: "development"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "-o", sbin/"blocky"
```

Comment:
```suggestion
    ldflags = %W[
      -s -w
      -X github.com/0xERR0R/blocky/util.Version=#{version}
      -X github.com/0xERR0R/blocky/util.BuildTime=#{time.iso8601}
    ]
    system "go", "build", *std_go_args(ldflags:ldflags, output: sbin/"blocky")
```

Reply:
The binary can runs in both client and server mode, so why not create the binary in `bin`, and symlink to sbin for server usage?

Reply:
You do need both version and buildtime for version output, unfortunately, it looks like architecture is not dynamically overridable

```
$ /opt/homebrew/Cellar/blocky/0.20/sbin/blocky version
blocky
Version: 0.20
Build time: 2022-11-12T20:39:38Z
Architecture: undefined
```

Reply:
A symlink was added for `sbin`.

Reply:
Architecture used to be set with `${ARCHITECTURE}` but now it's `${GOARCH}${GOARM}`.

https://github.com/0xERR0R/blocky/commit/e6dd093459f229a4fe44be7039655634a04af97a

Reply:
We should setup those build environment variables.